packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { forwardRef, useMemo } from 'react';
+import Barcode from 'react-barcode';
+import { useTranslation } from 'react-i18next';
+import { getPatientName, showSnackbar, useConfig, getCoreTranslation } from '@openmrs/esm-framework';
+import { defaultBarcodeParams, getPatientField } from './print-identifier-sticker.resource';
+import { type ConfigObject } from '../config-schema';
+import styles from './print-identifier-sticker.scss';
+
+interface PrintComponentProps {
+  patient: fhir.Patient;
+}
+
+const PrintComponent: React.FC<PrintComponentProps> = ({ patient }) => {
+  const { printPatientSticker } = useConfig<ConfigObject>();
+
+  const primaryIdentifierValue = patient?.identifier?.find((identifier) => identifier.use === 'official')?.value;
+  return (
+    <div className={styles.stickerContainer}>
+      <div className={styles.documentHeader}>
+        {printPatientSticker?.header?.showBarcode && (
+          <Barcode value={primaryIdentifierValue} {...defaultBarcodeParams} />
+        )}
+        {printPatientSticker?.header?.showLogo && (
+          <div className={styles.implementationLogo}>
+            <ImplementationLogo />
+          </div>
+        )}
+      </div>
+      <div className={styles.fieldsContainer}>
+        {printPatientSticker.fields.map((field) => {
+          const Component = getPatientField(field);
+          return (
+            <div key={field} className={styles.fieldRow}>
+              <Component patient={patient} />
+            </div>
+          );
+        })}
+      </div>
+    </div>
+  );
+};
+
+const ImplementationLogo: React.FC = () => {
+  const { t } = useTranslation();
+  const { printPatientSticker } = useConfig<ConfigObject>();
+
+  return printPatientSticker?.header?.logo ? (
+    <img alt={t('implementationLogo', 'Implementation logo')} src={printPatientSticker?.header?.logo} />
+  ) : (
+    // TODO: Figure out why #omrs-logo-full-mono sprite is not working
+    <svg data-testid="openmrs-logo" role="img" viewBox="0 0 380 119" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
    <svg aria-label={t('identifierStickerImplementationLogo', 'Identifier sticker implementation logo')} role="img" viewBox="0 0 380 119" xmlns="http://www.w3.org/2000/svg">
```

nit: Consider using an `aria-label` here over a test ID for more robustness.